src/frontend/Sider/Behandling/Inngangsvilkår/Målgruppe/EndreMålgruppeRad.tsx

```diff
@@ -152,7 +154,11 @@ const EndreMålgruppeRad: React.FC<{
                     form={form}
                     oppdaterTypeIForm={oppdaterType}
                     oppdaterPeriode={oppdaterForm}
-                    typeOptions={målgruppeTypeOptions}
+                    typeOptions={
+                        behandling.stønadstype === Stønadstype.LÆREMIDLER
```

Comment:
Hadde kanskje vært fint med en switch for disse, muligens noe som definierte typer som skulle med for gitt stønadstype
Jeg laget det sånn her for årsak avslag
https://github.com/navikt/tilleggsstonader-sak-frontend/commit/1718018d58b7507b851b35f709378bbfdb190361#diff-3d2d12596e074a65f8a59863976ae0147e0719d2e11934c3e84870158da75475R51

Reply:
Fixed: 0f520895b1d0ae63fe4aa3a747407f9447517018

Reply:
Fixed: [5d45be2](https://github.com/navikt/tilleggsstonader-sak-frontend/pull/633/commits/5d45be2314afb96a0d399a116d1df89f428b9825)